operator/config/manifests/bases/rhacs-operator.clusterserviceversion.yaml

```diff
@@ -718,6 +718,16 @@ spec:
 
     ## Using the RHACS Operator
 
+    **RHACS Operator channels:**
```

Comment:
@lmaynard88 @kcarmichael08 Could you please check my writing in this file? The contents are displayed to OpenShift users through the OperatorHub UI therefore I want to be extra careful that this content makes sense and follows Red Hat guidelines.

Reply:
@kcarmichael08 thanks for you review. I applied your suggestions plus added couple more minor changes. Could you please take another look?